packages/astro/src/runtime/client/dev-toolbar/apps/audit/rules/a11y.ts

```diff
@@ -107,6 +107,9 @@ const aria_non_interactive_roles = [
 	'tooltip',
 ];
 
+// These elements aren't interactive and aren't non-interactive. Their interaction changes based on the role assigned to them
+const roleless_elements = ['div', 'span'];
```

Comment:
Is there a documentation we can link to here?

Reply:
Done in [`69dc9a8` (#10719)](https://github.com/withastro/astro/pull/10719/commits/69dc9a8325328825dfe14390003455babef3e06b)

Reply:
Awesome, thank you